### ISSUE TITLE
Custom path for storing session.json(es)

### ISSUE THREAD

**Issue by JoeDozan24** [open-wa/wa-automate-nodejs#448]
Hello smashah,
Can we request a custom path for session.json(es) folder? To make it more tidier.
Thank you.

**Comment by smashah** [comment]
yeah good idea

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.31](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.30...1.9.31) (2020-05-19)

- fix: broken methods widfactory fixed #447 [`#447`](https://github.com/open-wa/wa-automate-nodejs/issues/447)
- Release 1.9.31 [`9a2d9e7`](https://github.com/open-wa/wa-automate-nodejs/commit/9a2d9e788a669551271f9f07ecb15326ef975d04)
- chore: fix name for config in initClient [`b2f9b63`](https://github.com/open-wa/wa-automate-nodejs/commit/b2f9b63a2d010d561a87b2b52ccbf9cc8e88ade9)
- Feature: implemented sessionDataPath #448 [`a079d5e`](https://github.com/open-wa/wa-automate-nodejs/commit/a079d5eede935fe4c3f18fe6460e282649afbdd8)
- updated release-image [`d7af129`](https://github.com/open-wa/wa-automate-nodejs/commit/d7af1292ba428019cc8335c0e96677d2a0ec75c5)

**Comment by smashah** [comment]
@JoeDozan24 https://open-wa.github.io/wa-automate-nodejs/interfaces/configobject.html#sessiondatapath
